static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
I don't understand when `System.normalize` should and shouldn't be applied. I assumed it was applied all the time?